stdlib/_operator.pyi

```diff
@@ -95,15 +95,15 @@ def length_hint(__obj: object, __default: int = 0) -> int: ...
 @final
 class attrgetter(Generic[_T_co]):
     @overload
-    def __new__(cls, attr: str) -> attrgetter[Any]: ...
+    def __new__(cls, __attr: str) -> attrgetter[Any]: ...
     @overload
-    def __new__(cls, attr: str, __attr2: str) -> attrgetter[tuple[Any, Any]]: ...
+    def __new__(cls, __attr: str, __attr2: str) -> attrgetter[tuple[Any, Any]]: ...
     @overload
-    def __new__(cls, attr: str, __attr2: str, __attr3: str) -> attrgetter[tuple[Any, Any, Any]]: ...
+    def __new__(cls, __attr: str, __attr2: str, __attr3: str) -> attrgetter[tuple[Any, Any, Any]]: ...
     @overload
-    def __new__(cls, attr: str, __attr2: str, __attr3: str, __attr4: str) -> attrgetter[tuple[Any, Any, Any, Any]]: ...
+    def __new__(cls, __attr: str, __attr2: str, __attr3: str, __attr4: str) -> attrgetter[tuple[Any, Any, Any, Any]]: ...
     @overload
-    def __new__(cls, attr: str, *attrs: str) -> attrgetter[tuple[Any, ...]]: ...
+    def __new__(cls, __attr: str, *attrs: str) -> attrgetter[tuple[Any, ...]]: ...
     def __call__(self, obj: Any) -> _T_co: ...
```

Comment:
While we're here:

```suggestion
    def __call__(self, obj: Any, /) -> _T_co: ...
```


```
In [33]: operator.attrgetter("count")(obj="x")
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[33], line 1
----> 1 operator.attrgetter("count")(obj="x")

TypeError: attrgetter() takes no keyword arguments

In [34]: operator.attrgetter("count")("x")
Out[34]: <function str.count>
```

(Also, we can use PEP 570 pos-only syntax now.)

Reply:
Nice! 👍